### DISCUSSION TITLE
Ideas about Android TV integration

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#1182] [General]
Hi!
Is there any thoughts on how to handle streaming images/videos to Android TV or chromecast (or other platforms). On Google Photos its possible to "cast" although it does not work very good. Now that I have all my pictures in folders I can also use SMB and a file browser in my Android TV box to browse all the pictures.
Is there any plans to have some app or interface to be able to show pictures (with albums) on android TV? Or some workaround in the mean time?

**Comment by alextran1502** [comment]
We talked about implementing it at some point but there is no defined plan yet. If you want, I encourage you to open an issue so we can keep track of it

**Reply by jagjordi** [reply]
Would it be possible to simply sideload the existing apk? Should it work? I can try that in a couple of days and report back

**Comment by Pheggas** [comment]
Please, close this one as #1614 is already open.
